src/tts_text/borger_dk.py

```diff
@@ -0,0 +1,269 @@
+"""Scraping and preprocessing of the borger.dk text corpus."""
+
+from pathlib import Path
+from unicodedata import normalize
+from bs4 import Tag, BeautifulSoup
+from omegaconf import DictConfig
+from tqdm.auto import tqdm
+from webdriver_manager.chrome import ChromeDriverManager
+import logging
+import re
+from .utils import extract_sentences, get_soup
+
+
+BASE_URL = "https://www.borger.dk"
+SUBSITES_TO_IGNORE = [
+    "/kampagnesider/",  # Its own special subsite, with a question-answer-type
+    # navigation, which leads to articles on the main site
+]
+
+
+def build_borger_dk_dataset(cfg: DictConfig) -> list[str]:
+    """Build the borger.dk dataset.
+
+    Args:
+        cfg: The Hydra configuration object.
+
+    Returns:
+        A list of articles from borger.dk.
+    """
+    # Load dataset if it already exists
+    dataset_path = Path(cfg.dirs.data) / cfg.dirs.raw / "borger_dk.txt"
+    if dataset_path.exists():
+        with dataset_path.open("r", encoding="utf-8") as f:
+            return f.read().split("\n")
+
+    # Install the Chrome driver, if it isn't already installed
+    logging.getLogger("WDM").setLevel(logging.WARNING)
+    ChromeDriverManager().install()
+
+    # Get the overall categories from the front page
+    # retry if it fails
+    soup: BeautifulSoup = BeautifulSoup("")
+    while not soup.contents:
+        soup = get_soup(url=BASE_URL, dynamic=True)
```

Comment:
Maybe include this into the `get_soup` function, so that this feature can be used for sundhed.dk as well?